GWAS/liftover.ipynb

```diff
@@ -62,7 +62,8 @@
     "- `--ouput_file`, the name of ouput file which will be saved under `cwd` path\n",
     "- `--fr`, From reference genome, defaut is `hg19`\n",
     "- `--to`,To reference genome, defaut is `hg38`\n",
-    "- `--remove-missing`, boolen, Remove SNPs failed to liftover (defaults to False)"
+    "- `--remove-missing`, boolen, Remove SNPs failed to liftover (default to False)\n",
+    "- `--no-rename`, boolen, Rename variants' ID (default to False). **Only implemented to sumstat liftover**"
```

Comment:
@changebio please use `parameter rename = True`. In the command interface, `--no-rename` sets this to `False`. This is a  SoS convention. Let's follow that standard.